src/Display/Tabs/Tabs.tsx

```diff
@@ -2,23 +2,18 @@ import * as React from 'react';
 
 import classNames from 'classnames';
 
-import { ETabThemeVariant } from '../../Utils/StyleConfig';
 import TabPane, { Props as TabPaneProps } from './TabPane';
 
 import { TabsContainer, TabsHeader, TabsBody } from './TabsStyle';
 
-export type Variant = 'underlined' | 'colored';
-export type Alignment = 'horizontal' | 'vertical';
-export type Theme = 'blue' | 'black';
-
-const Tabs: Tabs = ({
+export const Tabs: Tabs = ({
   activeTab,
   onTabClick,
   children,
   variant = 'underlined',
   className,
   alignment = 'horizontal',
-  theme = ETabThemeVariant.BLACK,
+  theme = 'black',
```

Comment:
Oh what is the issue of using enum ETabThemeVariant?

Reply:
It didn't play nice with the args table.

![Screenshot from 2021-01-12 10-12-26](https://user-images.githubusercontent.com/3341028/104260609-cf196700-54be-11eb-98b6-17523a6dddf4.png)

Reply:
Ah I see, I think can try to use `as const` instead of `enum`, but I'm ok with this. Because TS should help us check the default value already.